### PR TITLE
fixup! .circleci: Update rtl8730e configurations

### PR DESCRIPTION
**Pull request by ritesh55555**
updates the rtl8730e configuration from hello to hello_8M and added loadable_apps_8M in .circleci/config.yml

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -177,6 +177,9 @@ workflows:
       - rtl8720e/loadable_apps:
           requires:
             - checkout_code
-      - rtl8730e/hello:
+      - rtl8730e/hello_8M:
+          requires:
+            - checkout_code
+      - rtl8730e/loadable_apps_8M:
```
Comment (abhishek-samsung) [inline .circleci/config.yml:183]: https://github.com/Samsung/TizenRT/blob/96bbade61ebc84551908bbc4aca0df7eff49c68e/.circleci/config.yml#L138

I think you will have to change the above line too in order to support the newly added config build.

**Comment by sunghan-chang** [comment]
@ritesh55555 Could you check @abhishek-samsung 's comment?

File: .circleci/config.yml (hunk 1 above)
Comment (ritesh55555) [reply .circleci/config.yml:183]: done

**Review by abhishek-samsung — APPROVED** [review]
looks good

**Review by sunghan-chang — APPROVED** [review]

**Pull request merged**